int16 cmdLeft
int16 cmdRight
